src/main/java/com/ait/lienzo/client/core/shape/wires/WiresConnector.java

```diff
@@ -731,6 +663,260 @@ public Point2DArray getControlPoints(){
         return m_line.getPoint2DArray();
     }
 
+    public int addControlPoint(final double x,
```

Comment:
The whole gigantic (289 lines of code) batch of code is not covered.

Reply:
Also found:

`return getGroup().uuid() == that.getGroup().uuid();`

`==` instead of `equals` method for Strings in `@Override public boolean equals(Object o)` method.